webapp/channels/src/components/post_view/scroll_to_bottom_arrows.tsx

```diff
@@ -5,31 +5,31 @@ import React from 'react';
 
 import ScrollToBottomIcon from 'components/widgets/icons/scroll_to_bottom_icon';
 
-interface Props {
+type Props = {
     isScrolling: boolean;
     atBottom?: boolean;
     onClick: () => void;
 }
 
-export default class ScrollToBottomArrows extends React.PureComponent<Props> {
-    render() {
-        // only show on mobile
-        if (window.innerWidth > 768) {
-            return null;
-        }
-
-        let className = 'post-list__arrows';
-        if (this.props.isScrolling && this.props.atBottom === false) {
-            className += ' scrolling';
-        }
+const ScrollToBottomArrows = ({ isScrolling, atBottom, onClick }:Props) => {
+    // only show on mobile
+    if (window.innerWidth > 768) {
+        return null;
+    }
 
-        return (
-            <div
-                className={className}
-                onClick={this.props.onClick}
-            >
-                <ScrollToBottomIcon/>
-            </div>
-        );
+    let className = 'post-list__arrows';
+    if (isScrolling && !atBottom) {
```

Comment:
Here there is a small change in behavior. If `atBottom` was only defined as a boolean, `atBottom===false` would be equivalent to `!atBottom`. But this one is `boolean | undefined`. So... `undefined === false` is not the same as `!undefined`.

Long story short...
```suggestion
    if (isScrolling && atBottom === false) {
```